src/app/home/cardsForMetaCampaigns.ts

```diff
@@ -53,6 +58,33 @@ export const cardsForMetaCampaigns = function (metacampaigns: readonly MetaCampa
     anyExploreCard = [];
   }
 
+  if (viewingDate >= new Date("2022-04-20T12:00:00+01:00")) {
+    return [
+      ...metaCampaignCards,
+      {
+        headerText: 'Double your donation in the Green Match Fund',
+        backgroundImageUrl: new URL('/assets/images/card-background-gmf.jpg', donateUriPrefix),
+        iconColor: 'brand-3',
+        bodyText: 'Donate between 20/04/23 - 27/04/23',
```

Comment:
I think this would look better if the date format matched the next card. We might also be able to omit "23"?

Reply:
Agreed, and generally the numbers only format is probably best avoided since its ambiguous for the first 12 days of the month. Also to me "between" reads as redundant with "-". We could change it to the below. @BethBigGive what do you think?

```suggestion
        bodyText: 'Donate from 20 April to 27 April 2023',
```

Reply:
Happy to make that change @bdsl

Reply:
![image](https://user-images.githubusercontent.com/159481/232814750-529fd3cb-59e0-455e-8cbe-1ef877ea91f1.png)